### PR TITLE
Instructions about updating existing projects' and assets

### PR DESCRIPTION
**Pull request by ironfroggy**

### CODE REVIEW

File: source/frontend-packages.rst [hunk 1]
```diff
@@ -40,13 +40,134 @@ You can download and add these from this documentation:
...
+    <script src="{% static 'libs/modernizr.js' %}"></script>
+
+jQuery
+''''''
+
+If you need the jQuery library accessible for script tags that might be in specific pages
+and won't be pulled in by the bundling process, then it might be prudent to keep the
+jquery library included by the existing ``<script>`` tag, and that's okay. You might
+keep the need to move this Javascript out of templates and into modules as part of
+improvements to make down the road.
```
Comment (vkurup) [inline source/frontend-packages.rst]: This last sentence is a little confusing to me. Maybe add 'for now' to the previous sentence. And then change the final sentence to 'You can move this JS out of the template in a future improvement.' or something similar.

**Comment by vkurup** [comment]
1 comment, but looks good otherwise. :shipit:

**Comment by copelco** [comment]
Is there a proper way to handle CSS from 3rd party libraries? For example, I've installed nvd3 through npm and imported it in site.js, is there an equivalent for the CSS side of things?

File: source/frontend-packages.rst [hunk 2]
```diff
@@ -40,13 +40,134 @@ You can download and add these from this documentation:
...
+'''''''''
+
+Your project likely includes a copy of Modernizr, but the new frontend setup builds
+a recent copy of this (and is configurable), so we can change this line::
+
+    <script src="//cdnjs.cloudflare.com/ajax/libs/modernizr/2.7.1/modernizr.dev.js"></script>
+
+to this::
+
+    <script src="{% static 'libs/modernizr.js' %}"></script>
```
Comment (dpoirier) [inline source/frontend-packages.rst]: Possibly a naive question, but why isn't this built into the bundle too, saving another network request to load this library?  (And maybe a note should be added here for others who might wonder?)

Comment (ironfroggy) [inline source/frontend-packages.rst]: We load `modernizr.js` in the header to let it inject feature classes into the body as soon as possible, avoiding flicker from showing elements that might be shown or hidden by the feature classes being injected.

We load `bundle.js` in the footer after the entire body should be parsed into the DOM before our application code and dependent libraries run and potentially access that DOM.

**Comment by ironfroggy** [comment]
@copelco I have doing Less `@import` statements directly out of `node_modules/` something like this:

```
@import "../../node_modules/nvd3/build/nv.d3.css";
```

This works today, but soon I'd like to look at adding these locations to the `lessc` import paths, but I don't think its super pertinent. There are also options around the use of Webpack, which is kind of the next generation of Browserify that can handle stylesheets as well as JS modules, potentially.

File: source/frontend-packages.rst (hunk 1 above)
Comment (ironfroggy) [inline source/frontend-packages.rst]: :heavy_check_mark:

**Pull request merged**